lib/Controller/PageController.php

```diff
@@ -243,6 +249,8 @@ public function index(string $token = '', string $callUser = '', string $passwor
 			}
 		}
 
+		$this->eventDispatcher->dispatchTyped(new LoadViewer());
```

Comment:
This does boom when the viewer app is not there. Should we wrap it with an isEnabledForUser check?

Reply:
Right, although I have used [the same approach used in the server](https://github.com/nextcloud/server/pull/19777/files#diff-e22e3f341fde43829c2e9275e48e6993R286) instead.